src/include/distributed/maintenanced.h

```diff
@@ -0,0 +1,18 @@
+/*-------------------------------------------------------------------------
+ *
+ * maintenanced.h
+ *
+ * Copyright (c) 2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
```

Comment:
Should also be in `shared_library_init.h`, and maybe have some meaningful comment.